server/src/core/odoo.rs

```diff
@@ -800,7 +800,7 @@ impl SyncOdoo {
      */
     pub fn get_symbol_of_opened_file(session: &mut SessionInfo, path: &PathBuf) -> Option<Rc<RefCell<Symbol>>> {
         let path_in_tree = path.to_tree_path();
-        for entry in session.sync_odoo.entry_point_mgr.borrow().iter_for_import(session.sync_odoo.entry_point_mgr.borrow().main_entry_point.as_ref().unwrap()) {
+        for entry in session.sync_odoo.entry_point_mgr.borrow().iter_main() {
             if (entry.borrow().typ == EntryPointType::MAIN || entry.borrow().addon_to_odoo_path.is_some()) && entry.borrow().is_valid_for(path) {
```

Comment:
Due to this `(entry.borrow().typ == EntryPointType::MAIN || entry.borrow().addon_to_odoo_path.is_some())` , we loop over public /builtin eps but always skip them. 

Also since iter_main chains the iterators, it will be just empty if the option is None, so it guards against unwrap errors and also sorts for performance